scipy/sparse/linalg/eigen/lobpcg/lobpcg.py

```diff
@@ -197,8 +197,7 @@ def lobpcg(A, X,
     It is not that ``n`` should be large for the LOBPCG to work, but rather the
     ratio ``n / m`` should be large. It you call LOBPCG with ``m=1``
     and ``n=10``, it works though ``n`` is small. The method is intended
-    for extremely large ``n / m``, see e.g., reference [28] in
-    https://arxiv.org/abs/0705.2626
+    for extremely large ``n / m``, see e.g. [2]_.
```

Comment:
Nit: 'see e.g. [2]_' sounds weird to me. (feel free to close this if it makes sense to you)

Reply:
looking at this edit a bit further, this is literally reference "[28]", which is indirect. Looking at the manuscript, this should be corrected to a new reference [doi:10.1145/1188455.1188504](https://doi.org/10.1145/1188455.1188504).